docs/eventing/samples/ping-source/README.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "PingSource example"
+linkTitle: "PingSource"
+weight: 10
+type: "docs"
+---
+
+PingSource example shows how to configure PingSource as event source for
+functions.
+
+## Before you begin
+
+1. Setup [Knative Serving](../../../serving).
+1. Setup [Knative Eventing](../../../eventing).
+
+## Create a Knative Service
+
+In order to verify `PingSource` is working, we will create a simple Knative
+Service that dumps incoming messages to its log.
+
+```yaml
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: event-display
+spec:
+  template:
+    spec:
+      containers:
+        - image: gcr.io/knative-releases/github.com/knative/eventing-sources/cmd/event_display
+```
+
+Use following command to create the service from `service.yaml`:
+
+```shell
+kubectl apply --filename service.yaml
+```
+
+## Create a PingSource
+
+For each set of ping events you want to request, you need to create an Event
+Source in the same namespace as the destination. If you need a different
+ServiceAccount to create the Deployment, modify the entry accordingly in the
+yaml.
```

Comment:
```suggestion
For each set of ping events that you want to request, create an Event
Source in the same namespace as the destination. If you need a different
service account to create the Deployment, modify the entry accordingly in the
YAML file.
```

Suggest changing ServiceAccount to the generic term, unless you specifically want to refer to the API object, in which case I suggest using code font.